indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -194,7 +207,15 @@
         insert-time (index-util/date->elastic insert-time)
         coordinate-system (get-in collection [:SpatialExtent :HorizontalSpatialDomain
                                                        :Geometry :CoordinateSystem])
-        permitted-group-ids (get-coll-permitted-group-ids context provider-id collection)]
+        permitted-group-ids (get-coll-permitted-group-ids context provider-id collection)
+        {:keys [granule-start-date granule-end-date]} (cgac/get-coll-gran-aggregates context concept-id)
+        last-3-days (t/interval (t/minus (tk/now) (t/days 3)) (tk/now))
+        granule-end-date (if (and granule-end-date (t/within? last-3-days granule-end-date))
```

Comment:
You could change this `if` to use `when-not`
https://clojuredocs.org/clojure.core/when-not